src/app/screens/Accounts/index.tsx

```diff
@@ -46,7 +60,21 @@ function AccountsScreen() {
     });
 
     getAccounts();
-    closeModal();
+    closeEditModal();
+  }
+
+  async function exportAccount({ id, name }: AccountAction) {
+    setLoading(true);
+    setTimeout(() => {
+      setExportModalIsOpen(true);
+    }, 50);
```

Comment:
Sorry, one last thing. Could you add the comment:

```
                            /**
                             * @HACK
                             * @headless-ui/menu restores focus after closing a menu, to the button that opened it.
                             * By slightly delaying opening the modal, react-modal's focus management won't be overruled.
                             * {@link https://github.com/tailwindlabs/headlessui/issues/259}
                             */
                            setTimeout(() => {
                              setModalIsOpen(true);
                            }, 50);
```

Eventually we want to get rid of this hack and need to find all occurrences 🤞

Reply:
Done! Ready to merge now!

Reply:
Btw there are three (including above) occurences if this hack: 
`src/app/components/AllowanceMenu/index.tsx` (1)
`src/app/screens/Accounts/index.tsx` (2)
And issue (https://github.com/tailwindlabs/headlessui/issues/259) doesn't seem to be resolved.
Gotta do something

Reply:
Problem is, that the issue is closed. Maybe we should open a new one with a codesandbox example or something?